src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: $font-weight-regular;
+  font-style: normal;
+}
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid setting global styles on html or body tags directly to prevent potential conflicts. Consider using a class for these styles.